demo/live_dashboard/run.py

```diff
@@ -3,6 +3,7 @@
 import datetime
 import plotly.express as px
```

Comment:
add a requirements.txt file in this repo with all the required packages so it works on spaces.

Reply:
nevermind thought it was a new demo